packages/react/src/header/Header.stories.tsx

```diff
@@ -98,3 +102,27 @@ export const WithMainNav: Story = {
 		</Fragment>
 	),
 };
+
+export const CoBranding: Story = {
```

Comment:
Should probably have a `dividerPositionBetween` story

Reply:
I never know where to draw the line. If the prop requires a primitive value, I usually just leave it to the addons 🤷  But, I'm happy to add one here.

- [ ] dividerPositionBetween story